doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -4159,6 +4161,10 @@ \subsubsection{Outputs}\label{outputs-10-005}
 
 This output is the operating part-load ratio of the boiler. The part-load ratio is calculated as the boiler load divided by the boiler nominal capacity. The part-load ratio is limited by the minimum and maximum part-load ratio inputs specified by the user.
 
+\paragraph{Boiler Efficiency {[]}}\label{boiler-efficiency}
+
```

Comment:
I like the note, but just for the sake of uniformity and completeness, we should probably also have an explicit definition of efficiency. Maybe add something like "this output is boiler heating rate divided by boiler gas rate" or similar between the label and the note.